### PR TITLE
Move the note to the top in GMT version migration

### PR DESCRIPTION
**Pull request by core-man**
**Preview:** https://gmt-china.github.io/sitepreview/gmt-china/GMT_docs/version-migration/migrating/

### CODE REVIEW

File: source/migrating/index.rst [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
+.. note::
 
-   4to5
-   5to6
-   classic2modern
+   本文仅限于GMT老用户以及需要将老脚本转换为新语法的新用户阅读。
+   纯粹的GMT新用户请跳过这一章以节省时间并避免可能出现的混淆。
 
 GMT在大版本升级时，语法及用法上会发生一些变化，但由于GMT尽可能兼容旧版本的语法。
 因而用户只需要少量修改即可在新版本GMT中运行旧版本GMT下写的脚本。
```
Comment (core-man) [inline source/migrating/index.rst:10]:
```suggestion
GMT 在大版本升级时，语法及用法上会发生一些变化。但由于 GMT 会尽可能兼容旧版本的语法，
用户往往只需要对旧版本 GMT 语法编写的脚本做少量修改，即可在新版本 GMT 中运行。
```

File: source/migrating/index.rst [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
-.. toctree::
-   :hidden:
-   :maxdepth: 2
+.. note::
 
-   4to5
-   5to6
-   classic2modern
+   本文仅限于GMT老用户以及需要将老脚本转换为新语法的新用户阅读。
+   纯粹的GMT新用户请跳过这一章以节省时间并避免可能出现的混淆。
```
Comment (core-man) [inline source/migrating/index.rst:7]:
```suggestion
   本文仅限于 GMT 老用户以及需要将老脚本转换为新语法的新用户阅读。
   纯粹的 GMT 新用户请跳过这一章以节省时间并避免可能出现的混淆。
```

File: source/migrating/index.rst [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
 
-   4to5
-   5to6
-   classic2modern
+   本文仅限于 GMT 老用户以及需要将老脚本转换为新语法的新用户阅读。
+   纯粹的 GMT 新用户请跳过这一章以节省时间并避免可能出现的混淆。
 
-GMT在大版本升级时，语法及用法上会发生一些变化，但由于GMT尽可能兼容旧版本的语法。
-因而用户只需要少量修改即可在新版本GMT中运行旧版本GMT下写的脚本。
+GMT 在大版本升级时，语法及用法上会发生一些变化。但由于 GMT 会尽可能兼容旧版本的语法，
```
Comment (wangliang1989) [inline source/migrating/index.rst:9]: 大版本升级不明确，何为大版本升级？

Comment (core-man) [inline source/migrating/index.rst:9]:
```suggestion
GMT 在大版本升级时，语法及用法上会发生一些变化，如 GMT 5 升级为 GMT 6。但由于 GMT 会尽可能兼容旧版本的语法，
```

Comment (core-man) [reply source/migrating/index.rst:9]: See the commit below.

File: source/migrating/index.rst [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+用户往往只需要对旧版本 GMT 语法编写的脚本做少量修改，即可在新版本 GMT 中运行。
```
Comment (core-man) [inline source/migrating/index.rst:10]:
```suggestion
但由于 GMT 会尽可能兼容旧版本的语法，用户往往只需要对旧版本 GMT 语法编写的脚本做少量修改，
即可在新版本 GMT 中运行。
```

**Review by seisman — APPROVED** [review]

**Pull request merged**